src/Symfony/Component/Mailer/Bridge/Sendgrid/Transport/SendgridApiTransport.php

```diff
@@ -125,7 +126,13 @@ private function getPayload(Email $email, Envelope $envelope): array
                 continue;
             }
 
-            if ($header instanceof TagHeader) {
+            if ($name === 'send-at') {
```

Comment:
```suggestion
            if ('send-at' === $name) {
```
Use Yoda conditions to comply with the coding standards:
https://symfony.com/doc/current/contributing/code/standards.html#structure